docs/source/concepts/SystemicVariation/index.rst

```diff
@@ -3,11 +3,10 @@
 Systemic Variation
 !!!!!!!!!!!!!!!!!!
 
-VRS currently covers many classes of variation that are defined on a contiguous molecule such as single nucleotide
-variants (SNVs), multi-nucleotide variants (MNVs), indels, repeats, haplotypes, breakpoints, and sequence
-rearrangements that form derivative molecules.
+VRS currently covers many classes of variation that are defined on multiple molecules
+in the context of a system, e.g. a genome, sample, or homologous chromosomes.
```

Comment:
@ahwagner @larrybabb can you provide with alternative sentence (or else it will show up twice)